adapters/http-base/src/main/java/org/eclipse/hono/adapter/http/AbstractVertxBasedHttpProtocolAdapter.java

```diff
@@ -1036,6 +1015,7 @@ protected final Future<CommandConsumer> createCommandConsumer(
                         responseReady.handle(Future.succeededFuture());
```

Comment:
This line here and the line
`commandHandlerDonePromise.future().onComplete(responseReady);`
below mean that `responseReady` is being completed twice.

(This doesn't lead to a `Result is already complete` exception because `PromiseImpl.handle` actually does a `tryComplete`.)

Reply:
yes, I've also noticed that